app/models/application_record.rb

```diff
@@ -8,7 +7,7 @@ module VERSION
         MAJOR = ActiveRecord::VERSION::MAJOR
       end
     end
-  end unless Rails.const_defined?("VERSION")
+  end if !defined?(::Rails) || !::Rails.const_defined?("VERSION")
```

Comment:
@agrare when I removed the require ^, it was failing on undefined const ```Rails```, the acts-as-taggable was probably requiring rails somewhere

Reply:
Interesting, this hack was added because the acts_as_tenant gem was using Rails without checking if it was defined first, nice catch.